engine/crates/engine-v2/src/sources/graphql/mod.rs

```diff
@@ -84,6 +85,16 @@ pub(crate) struct GraphqlExecutor<'ctx> {
 impl<'ctx> GraphqlExecutor<'ctx> {
     #[tracing::instrument(skip_all, fields(plan_id = %self.plan.id(), federated_subgraph = %self.subgraph.name()))]
     pub async fn execute(mut self) -> ExecutionResult<ResponsePart> {
+        let operation_name = self.plan.operation().as_ref().name.as_ref();
+        let operation_type = self.plan.operation().as_ref().ty.as_ref();
+
+        let subgraph_request_span = grafbase_tracing::span::subgraph::SubgraphRequestSpan::new(self.subgraph.name())
```

Comment:
It's a bit long line, maybe `use` would be better for the `SubgraphRequestSpan`?